skimage/filters/thresholding.py

```diff
@@ -330,10 +330,10 @@ def threshold_yen(image, nbins=256):
     ----------
     .. [1] Yen J.C., Chang F.J., and Chang S. (1995) "A New Criterion
            for Automatic Multilevel Thresholding" IEEE Trans. on Image
-           Processing, 4(3): 370-378
+           Processing, 4(3): 370-378. DOI:10.1109/83.366472
```

Comment:
Hmm.. Here and below there is a syntax inconsistence: `.`, `,`, ` ` before the DOI.